File: app/src/main/java/com/github/kongpf8848/animation/activity/ViewPagerActivity.java
package com.github.kongpf8848.animation.activity;

import android.view.View;

import com.github.kongpf8848.animation.R;
import com.github.kongpf8848.animation.activity.property.BallActivity;
import com.github.kongpf8848.animation.activity.property.CommentActivity;
import com.github.kongpf8848.animation.activity.property.PropertyDemoActivity;
import com.github.kongpf8848.animation.activity.property.SwapActivity;
import com.github.kongpf8848.animation.activity.viewpager.WeChatActivity;
import com.github.kongpf8848.animation.activity.viewpager.XhsActivity;

import butterknife.OnClick;


public class ViewPagerActivity extends BaseActivity {

    @Override
    public int getLayoutId() {
        return R.layout.activity_viewpager;
    }


    @OnClick(R.id.button1)
    public void onButton1(View view) {
        startActivity(WeChatActivity.class);
    }

    @OnClick(R.id.button2)
    public void onButton2(View view) {
        startActivity(XhsActivity.class);
    }



}

File: app/src/main/java/com/github/kongpf8848/animation/activity/ViewPagerActivity.kt
package com.github.kongpf8848.animation.activity

import android.view.View
import butterknife.OnClick
import com.github.kongpf8848.animation.R
import com.github.kongpf8848.animation.activity.viewpager.WeChatActivity
import com.github.kongpf8848.animation.base.BaseToolbarActivity

class ViewPagerActivity : BaseToolbarActivity() {

    public override fun getLayoutId(): Int {
        return R.layout.activity_viewpager
    }

    @OnClick(R.id.button1)
    fun onButton1(view: View?) {
        startActivity(WeChatActivity::class.java)
    }

    @OnClick(R.id.button2)
    fun onButton2(view: View?) {
        startActivity(SparkSplashActivity::class.java)
    }

}